charts/super-agent-deployment/templates/uninstall-job.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.cleanupManagedResources -}}
+{{- $uninstallJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "uninstall-job" ) -}}
+{{- /*
+The resources managed by the super-agent and the label selector are hardcoded on the super-agent.
+*/ -}}
+{{- $saCRList := (list "helmreleases.helm.toolkit.fluxcd.io" "helmrepositories.source.toolkit.fluxcd.io") -}}
+{{- $saResourcesLabelSelector := "app.kubernetes.io/managed-by=newrelic-super-agent" -}}
+{{- /*
+To understand why this job installs manifests instead of using Helm, read the comment at `job-manifests.yaml`.
+*/ -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    helm.sh/hook-weight: "10010"
+  name: {{ $uninstallJobName }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  ttlSecondsAfterFinished: 120
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ include "newrelic.common.serviceAccount.name" . }}
+      containers:
+        - name: delete-managed-resources
+          image: bitnami/kubectl
+          command:
+            - bash
+          args:
+            - -c
+            - |
+              set -o pipefail
+              {{- /* Delete the standard resources (configmaps) managed by the super-agent */ -}}
+              {{ $managedResourcesNamespace := .Release.Namespace }}
```

Comment:
I think we could use directly .Release.Namespace, I do not see a real benefit, but if you see it easier to read I am fine keeping it 😄

Reply:
I added the variable to deal with the scope of the `range` below, but I've just realized we can simply use `$.Release.Namespace` there. Thanks! [0960eac](https://github.com/newrelic/helm-charts/pull/1278/commits/0960eac689244d46c693a19881e3beed64f135cc)